apps/web/public/static/locales/en/common.json

```diff
@@ -431,7 +431,7 @@
   "password_hint_min": "Minimum 8 characters long",
   "password_hint_admin_min": "Minimum 15 characters long",
   "password_hint_num": "Contain at least 1 number",
-  "invalid_password_hint": "The password must be a minimum of 7 characters long containing at least one number and have a mixture of uppercase and lowercase letters",
+  "invalid_password_hint": "The password must be a minimum of 15 characters long containing at least one number and have a mixture of uppercase and lowercase letters",
```

Comment:
the 15 character minimum limit is only for the admin role. for other 7 characters are still allowed. maybe just check if the user role is admin then show the 15 character password hint else 7 chars.

Reply:
i'll update those changes 🙌🏼 . do we still have the number, uppercase and uppercase requirement for a normal user ?

Reply:
i'm planning on to convert the hardcoded number into template variable.

Reply:
check lib/auth

export function validPassword(password: string) {
  if (password.length < 7) return false;

  if (!/[A-Z]/.test(password) || !/[a-z]/.test(password)) return false;

  if (!/\d+/.test(password)) return false;

  return true;
}